microprofile-jwt/src/main/java/org/jboss/eap/qe/microprofile/jwt/tools/JwtClaims.java

```diff
@@ -0,0 +1,124 @@
+package org.jboss.eap.qe.microprofile.jwt.tools;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * JWT claims as described in specification. See https://github.com/eclipse/microprofile-jwt-auth/releases
+ */
+public class JwtClaims {
```

Comment:
again :) Why do we need this file?

Isn't available any standard class for this ?

How will customers do this stuff ? If each of them will do the same WF/EAP should provide it for them.

Reply:
https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/38#discussion_r350114220